<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>749</width>
    <height>1086</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 14pt &quot;Liberation Serif&quot;;</string>
  </property>
  <widget class="QPushButton" name="pushButton_finalback">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>800</y>
     <width>221</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Liberation Serif</family>
     <pointsize>14</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    color: white;
    border: 2px solid #555;
    border-radius: 20px;
    border-style: outset;
    background: qradialgradient(
        cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
        radius: 1.35, stop: 0 #fff, stop: 1 #888
        );
    padding: 5px;
    }

QPushButton:hover {
    background: qradialgradient(
        cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
        radius: 1.35, stop: 0 #fff, stop: 1 #bbb
        );
    }

QPushButton:pressed {
    border-style: inset;
    background: qradialgradient(
        cx: 0.4, cy: -0.1, fx: 0.4, fy: -0.1,
        radius: 1.35, stop: 0 #fff, stop: 1 #ddd
        );
    }</string>
   </property>
   <property name="text">
    <string>Züruck</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="fgroupBox_pInfo">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>711</width>
     <height>711</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Liberation Serif</family>
     <pointsize>14</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>ForbiddenCursor</cursorShape>
   </property>
   <property name="title">
    <string>Patientinformationen</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QComboBox" name="fcomboBox_gender">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <item>
       <property name="text">
        <string>Männlich</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Weiblich</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Divers</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="9" column="1">
     <widget class="QComboBox" name="fcomboBox_nLength">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <item>
       <property name="text">
        <string>10</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>20</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>30</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>40</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>50</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>60</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>70</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>80</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>90</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>100</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="13" column="1">
     <widget class="QLineEdit" name="flineEdit_wDepth">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLineEdit" name="flineEdit_firstname">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="15" column="1">
     <widget class="QLineEdit" name="flineEdit_nStitches">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="16" column="0">
     <widget class="QLabel" name="label_wSize_2">
      <property name="text">
       <string>Koornationsinfo (mm):</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="15" column="0">
     <widget class="QLineEdit" name="flineEdit_wSize">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QDateEdit" name="fdateEdit_birthdate">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="flineEdit_lastname">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="flineEdit_patientid">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="14" column="0">
     <widget class="QLabel" name="label_wSize">
      <property name="text">
       <string>Wundgröße (mm):</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="flabel_lastname">
      <property name="text">
       <string>Nachname:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="flabel_birthdate">
      <property name="text">
       <string>Geburtsdatum</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="12" column="0">
     <widget class="QLabel" name="label_wLocation">
      <property name="text">
       <string>Wundposition (mm):</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="flabel_emergency">
      <property name="text">
       <string>Notfallkontaktinformationen:</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="flabel_scheduledate">
      <property name="text">
       <string>geplantes Operationensdatum:</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLabel" name="flabel_patientid">
      <property name="text">
       <string>Vericherungs-ID des Patienten:</string>
      </property>
     </widget>
    </item>
    <item row="11" column="0">
     <widget class="QComboBox" name="fcomboBox_nDiameter">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <item>
       <property name="text">
        <string>FineNeedle: Gauge 30-33</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>StandardNeedle: Gauge 25-27</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>SutureNeedle: Gauge 18-22</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>SpinalNeedle: Gauge 16-25</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Biopsy Cutting Needle: Gauge 14-20</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="10" column="1">
     <widget class="QLabel" name="flabel_nPatient">
      <property name="text">
       <string>Eindringtiefe der Nadel:</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QLineEdit" name="flineEdit_notes">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="13" column="0">
     <widget class="QComboBox" name="fcomboBox_wLocation">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <item>
       <property name="text">
        <string>Head/Neck</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Chest/Abdomen</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Back</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Half-Curved Needle</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Arm</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Leg</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Hand</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Foot</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Groin</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Buttocks</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Face</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="11" column="1">
     <widget class="QLineEdit" name="flineEdit_nPatientDepth">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QDateEdit" name="fdateEdit_scheduledate">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="9" column="0">
     <widget class="QComboBox" name="fcomboBox_nTip">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <item>
       <property name="text">
        <string>Straight Needle</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>RoundBody Needle</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Half-Curved Needle</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Triangle Needle</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Reverse Cutting Needle</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Taper Point Needle</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Atraumatic Needle</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="14" column="1">
     <widget class="QLabel" name="label_nStitches">
      <property name="text">
       <string>Anzahl der Stichpunkte:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="12" column="1">
     <widget class="QLabel" name="label_wDepth">
      <property name="text">
       <string>Wundtiefe (mm):</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLabel" name="flabel_notes">
      <property name="text">
       <string>Notizen:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="flabel_firstname">
      <property name="text">
       <string>Vorname:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="flabel_Gender">
      <property name="text">
       <string>Geschlecht:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="16" column="1">
     <widget class="QTableWidget" name="co_tableWidget">
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">QTableWidget {
background-color: #f5f5f5;
color: #333333;
font-family: Arial;
font-size: 12px;
}

QTableWidget::item {
padding: 10px;
border-bottom: 1px solid #cccccc;
selection-background-color: #a0c8ff;
}

QTableWidget::item:selected {
background-color: #a0c8ff;
color: #ffffff;
}

QHeaderView::section {
background-color: #e0e0e0;
color: #333333;
font-weight: bold;
padding: 5px;
border-bottom: 1px solid #cccccc;
}

QTableWidget::horizontalHeader {
background-color: #e0e0e0;
color: #333333;
font-weight: bold;
border: none;
border-bottom: 2px solid #cccccc; /* Hinzufügen einer unteren Trennlinie */
}

QTableWidget::horizontalHeader::section {
padding: 5px;
border-bottom: 1px solid #cccccc;
}

QTableWidget::verticalHeader {
background-color: #e0e0e0;
color: #333333;
font-weight: bold;
border: none;
border-right: 2px solid #cccccc; /* Hinzufügen einer rechten Trennlinie */
}

QTableWidget::verticalHeader::section {
padding: 5px;
border-right: 1px solid #cccccc;
}</string>
      </property>
      <row>
       <property name="text">
        <string>1</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>2</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>3</string>
       </property>
      </row>
      <column>
       <property name="text">
        <string>X</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Y</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Z</string>
       </property>
      </column>
     </widget>
    </item>
    <item row="10" column="0">
     <widget class="QLabel" name="flabel_nDiameter">
      <property name="text">
       <string>Nadelstärke:</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QLabel" name="flabel_nTip">
      <property name="text">
       <string>Nadelspitze:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QLineEdit" name="flineEdit_emergency">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QLabel" name="flabel_nTip_2">
      <property name="text">
       <string>Nadellänge:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="flabel_heading">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>-30</y>
     <width>591</width>
     <height>121</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>85</red>
         <green>170</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>85</red>
         <green>170</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>190</red>
         <green>190</green>
         <blue>190</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="font">
    <font>
     <family>Liberation Serif</family>
     <pointsize>20</pointsize>
     <weight>3</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25 20pt &quot;Liberation Serif&quot;
</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600; color:#21a1c0;&quot;&gt;Patientenakte&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_newpatient">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>800</y>
     <width>221</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Liberation Serif</family>
     <pointsize>14</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    color: white;
    border: 2px solid #555;
    border-radius: 20px;
    border-style: outset;
    background: qradialgradient(
        cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
        radius: 1.35, stop: 0 #fff, stop: 1 #888
        );
    padding: 5px;
    }

QPushButton:hover {
    background: qradialgradient(
        cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
        radius: 1.35, stop: 0 #fff, stop: 1 #bbb
        );
    }

QPushButton:pressed {
    border-style: inset;
    background: qradialgradient(
        cx: 0.4, cy: -0.1, fx: 0.4, fy: -0.1,
        radius: 1.35, stop: 0 #fff, stop: 1 #ddd
        );
    }</string>
   </property>
   <property name="text">
    <string>neuer Patient</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_exit">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>920</y>
     <width>241</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Liberation Serif</family>
     <pointsize>14</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#pushButton_exit {
    color: white;
    border: 2px solid #555;
    border-radius: 20px;
    border-style: outset;
    background: qradialgradient(
        cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
        radius: 1.35, stop: 0 #fff, stop: 1 #888
    );
    padding: 5px;
}

QPushButton#pushButton_exit:hover {
    background: qradialgradient(
        cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
        radius: 1.35, stop: 0 #fff, stop: 1 #bbb
    );
}

QPushButton#pushButton_exit:pressed {
    border-style: inset;
    background: qradialgradient(
        cx: 0.4, cy: -0.1, fx: 0.4, fy: -0.1,
        radius: 1.35, stop: 0 #fff, stop: 1 #ddd
    );
}

QPushButton#pushButton_exit {
    background-color: #21a1c0;
}
</string>
   </property>
   <property name="text">
    <string>Beenden</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_saveAllinfo">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>800</y>
     <width>221</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Liberation Serif</family>
     <pointsize>14</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    color: white;
    border: 2px solid #555;
    border-radius: 20px;
    border-style: outset;
    background: qradialgradient(
        cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
        radius: 1.35, stop: 0 #fff, stop: 1 #888
        );
    padding: 5px;
    }

QPushButton:hover {
    background: qradialgradient(
        cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
        radius: 1.35, stop: 0 #fff, stop: 1 #bbb
        );
    }

QPushButton:pressed {
    border-style: inset;
    background: qradialgradient(
        cx: 0.4, cy: -0.1, fx: 0.4, fy: -0.1,
        radius: 1.35, stop: 0 #fff, stop: 1 #ddd
        );
    }</string>
   </property>
   <property name="text">
    <string>speichern</string>
   </property>
  </widget>
  <widget class="QLabel" name="flabel_nDiameter_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>990</y>
     <width>491</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 8pt &quot;Ubuntu&quot;;</string>
   </property>
   <property name="text">
    <string>© 2023 AS Interactive HealthTech. Alle Rechte vorbehalten.</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>-10</y>
     <width>241</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>logo.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_info">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>860</y>
     <width>301</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Liberation Serif</family>
     <pointsize>14</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    color: white;
    border: 2px solid #555;
    border-radius: 20px;
    border-style: outset;
    background: qradialgradient(
        cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
        radius: 1.35, stop: 0 #fff, stop: 1 #888
        );
    padding: 5px;
    }

QPushButton:hover {
    background: qradialgradient(
        cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
        radius: 1.35, stop: 0 #fff, stop: 1 #bbb
        );
    }

QPushButton:pressed {
    border-style: inset;
    background: qradialgradient(
        cx: 0.4, cy: -0.1, fx: 0.4, fy: -0.1,
        radius: 1.35, stop: 0 #fff, stop: 1 #ddd
        );
    }</string>
   </property>
   <property name="text">
    <string>Alle Patientendaten anzeigen</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
